wmo-im/bufr2geojson
# bootstrapping updates

- implement tests as pytest
- hookup testing to GitHub Actions CI
- add GitHub Container registry to GitHub Actions CI
- update README with install/run/release workflow
- add flake8 fixes
- cleanup JSON schemas
- update transform workflow to accept byte string (note: needs ecCodes fix)

Note that the jsonschema validation uses a local / patched jsonschema codebase that is able to work with remote references that are YAML.  The schema validation is quite slow (as can be seen in the CI); so some work will be needed to optimize/caching remote schemas.
